pylxd/exceptions.py

```diff
@@ -37,6 +40,17 @@ class LXDAPIExtensionNotAvailable(Exception):
     """An exception raised when requested LXD API Extension is not present
     on current host."""
 
+    def __init__(self, name, *args, **kwargs):
+        """Custom exception handling of the message is to convert the name in
```

Comment:
"in to" should read "into"

Reply:
Good catch; I'm always getting 'in to' and 'into' confused.  This was useful to me: https://blog.oxforddictionaries.com/2015/08/10/in-to-versus-into/